changes/4786.housekeeping

```diff
@@ -0,0 +1 @@
+Make buttons with class of `btn-default` background transparent and lighten table row background color for dark theme.
```

Comment:
This isn't `housekeeping`, as it's a user-facing change. I'd suggest the following:

- changes/4786.fixed: `Fixed default button background color in dark mode.`
- changes/4786.changed: `Lightened table row background color in dark mode.`